### DISCUSSION TITLE
2.3.80 ManagerSearch NODE Fresh Install Git Install CENTOS7  so_elastic_user are missing

### DISCUSSION THREAD

**Discussion by JAEP2** [Security-Onion-Solutions/securityonion#5823] [Unsupported Versions] [answered]
Dear all,

After a lot of trys, always I received the following errors during setup.

To reproduce it.

Fresh Install from GIT in CENTOS7 with 16GB RAM 400 SSD Disk  6 vCPU Cores
ManagerSearch Node

sosetup ERRORS
`[ERROR   ] Command 'so-user' failed with return code: 1
[ERROR   ] stdout: Database file does not exist yet, installation is likely not yet complete.
[ERROR   ] stderr: parse error: Invalid numeric literal at line 1, column 3
[ERROR   ] retcode: 1
[ERROR   ] {'pid': 28699, 'retcode': 1, 'stdout': "Database file does not exist yet, installation is likely not yet complete.\nSyncing users and roles between SOC and Elastic...\nElastic auth credentials for system user 'so_elastic_user' are missing", 'stderr': 'parse error: Invalid numeric literal at line 1, column 3\nparse error: Invalid numeric literal at line 1, column 3'}
[ERROR   ] Source file salt://elasticsearch/files/users not found in saltenv 'base'
[ERROR   ] Source file salt://elasticsearch/files/users_roles not found in saltenv 'base'
[ERROR   ] {'container_id': {'added': '8be24fd8017d61299d9466e45696a4afcd2390596ebdfdd59aeefff908e2b5e2'}}
`
As a result, the setup screen is always stopped at 64%   "Applying elasticsearch Salt state"
install from ISO is not possible due is a VPS.

Thanks you for your comments will be apreciated... 

KInd regards

(Finally I can finished the installation without error, using git  with versión 2.3.50) Now I have planned upgrade it to 2.3.80

**Comment by dougburks** [comment]
I just tested and it worked fine for me:
![image](https://user-images.githubusercontent.com/1659467/136960401-c243d259-f150-42e6-8fb3-51da70a53a37.png)

Do you have any firewalls or other devices that may be interfering with installation? If so, can you try bypassing them or try on another network?

Is there anything special about the username or password that you set?  If so, can you try using a simple username and password to see if that makes any difference?

**Comment by JAEP2** [comment, marked as answer]
Hi Doug,

Thanks you for your tests.

Finally after 4 fresh installs installed from 2.3.50 and upgraded with soup,  used in all installations the same usermane and password without any special chars,  in all cases the manager was installed in a VPS with fresh Centos7 with no firewalls  and direct internet connection. Only works for me with 2.3.50. 

I will try again as soon I can, i will update the results.

Thanks again for your reply.

Kind regards

**Reply by JAEP2** [reply]
Hi Doug
I just tested again from git version 2.3.80 and installed  manager search node without any issue, in the same machine with the same params.
if you want ,you can delete this thread.  Sorry by the disturbation.

the only difference today is that, I did not updated de Centos7 before  run the so-setup-network, the So script updated the OS today.

I will try to find the root cause of the error in the latest failed installations.

Kind regards
